documentation/decisions/adr/chat-adr

```diff
@@ -0,0 +1,128 @@
+| Threaded replies                                |                                                                                     ✅                                                                                     |                                  ❌Supports quoting                                 |                                                          ✅                                                         |                                                                  ✅                                                                 |                                                           ✅                                                           |                                                               ✅                                                               |                                             ✅                                             |
+| Emoji reactions                                 |                                                                                     ✅                                                                                     |                                          ✅                                         |                                                          ✅                                                         |                                                       ❌ Text based reactions                                                       |                                                           ✅                                                           |                                                               ✅                                                               |                                             ✅                                             |
+| Cost                                            | $12.50 USD per active user per month billed annually.Or, $15 USD per active user per month, billed monthly.Note: USWDS uses SlackFOSS community users must request access |         Free to run, free to use (see https&#x3A;//discord.com/open-source)        |                                  Part of MS365 subscription plans$12.50/user/month                                 |                                               $3.67 per user/month, billed annually;                                               | Free + own hosting costs; paid version available $7/user/monthMinimum of 25 usersor$35/agent/monthMinimum of 5 agents | free + own hosting costs; paid version available $10 per user/month, billed annually; enterprise pricing with requested quote | Free + own hosting costs; paid version available$6.67 per user per month, billed annually |
+
```

Comment:
used a markdown converter for this format, may change the view of the table so it's more clear and easier to edit in the future if needed.